### DISCUSSION TITLE
What Is The Best Nation Unique For Reimu?

### DISCUSSION THREAD

**Discussion by Wheelchair2** [Wheelchair2/Reimu-Hakurei-Touhou-Mod#1] [Q&A]
I was thinking what's the best nation unique for Reimu, since "+30% generation of Great Person" sounds a bit boring.

**Comment by hachchch** [comment]
How about "Free Great Person upon entering a Golden Age"？

**Reply by Wheelchair2** [reply]
Great idea, I might as well add a unique that makes all units with komainu guardian ability promotion get +1 movement in friendly territory during golden age
